geoips/interfaces/base.py

```diff
@@ -662,14 +666,14 @@ def get_plugin(self, name):
                 f" attempting to access the correct plugin name)"
             ) from resp
         # Convert the module into an object
-        return plugin_module_to_obj(name, module)
+        return plugin_module_to_obj(self, name, module)
 
     def get_plugins(self):
         """Get a list of plugins for this interface."""
         plugins = []
         for ep in get_all_entry_points(self.name):
             try:
-                plugins.append(plugin_module_to_obj(ep.name, ep))
+                plugins.append(plugin_module_to_obj(self, ep.name, ep))
```

Comment:
If we move plugin_module_to_obj onto this class as a classmethod, this line can be reverted.

Reply:
I'm going to create an issue for this. I don't want to include too many changes in this PR, and I feel that it would be best to have this separated in another branch. That way, if anything goes wrong, we can track it accordingly.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/369